dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_AnnotationLiteralsTest/server.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing the Faces 4.0 annotation literals">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <!-- Test differences between myfaces and mojarra if needed -->
+        <!-- <feature>facesContainer-4.0</feature> -->
+        <feature>faces-4.0</feature>
+        <feature>cdi-4.0</feature>
+        <feature>websocket-2.1</feature>
+    </featureManager>
+    
+    <javaPermission codeBase="${server.config.dir}/dropins/AnnotationLiteralsTest.war" className="java.security.AllPermission"/>
```

Comment:
What is the j2sec issue that prompted the addition of this?

Reply:
Here is the stack: 
```txt
[2/2/23, 14:27:09:419 CST] 0000004f kernel.launch.internal.MissingDoPrivDetectionSecurityManager W CWWKE0921W: Current Java 2 Security policy reported a potential violation of Java 2 Security Permission. The application needs to have permissions addedPermission: 
("java.lang.RuntimePermission" "accessDeclaredMembers")
Stack: 
java.security.AccessControlException: Access denied ("java.lang.RuntimePermission" "accessDeclaredMembers")java.base/java.security.AccessController.throwACE(AccessController.java:176)
java.base/java.security.AccessController.checkPermissionHelper(AccessController.java:238)
java.base/java.security.AccessController.checkPermission(AccessController.java:385)
java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:322)
com.ibm.ws.kernel.launch.internal.MissingDoPrivDetectionSecurityManager.checkPermission(MissingDoPrivDetectionSecurityManager.java:47)
java.base/java.lang.Class.checkMemberAccess(Class.java:266)
java.base/java.lang.Class.getDeclaredConstructor(Class.java:800)
jakarta.faces.FactoryFinder.addBeanEntry(FactoryFinder.java:443)
jakarta.faces.FactoryFinder.injectAndPostConstruct(FactoryFinder.java:379)
jakarta.faces.FactoryFinder.newFactoryInstance(FactoryFinder.java:485)
jakarta.faces.FactoryFinder._getFactory(FactoryFinder.java:327)
jakarta.faces.FactoryFinder.getFactory(FactoryFinder.java:207)
org.apache.myfaces.view.facelets.impl.DefaultFaceletFactory.<init>(DefaultFaceletFactory.java:95)
org.apache.myfaces.view.facelets.FaceletViewDeclarationLanguage.createFaceletFactory(FaceletViewDeclarationLanguage.java:2084)
org.apache.myfaces.view.facelets.FaceletViewDeclarationLanguage.<init>(FaceletViewDeclarationLanguage.java:242)
org.apache.myfaces.view.facelets.FaceletViewDeclarationLanguageStrategy.<init>(FaceletViewDeclarationLanguageStrategy.java:52)
org.apache.myfaces.view.ViewDeclarationLanguageFactoryImpl.initialize(ViewDeclarationLanguageFactoryImpl.java:95)
org.apache.myfaces.view.ViewDeclarationLanguageFactoryImpl.getViewDeclarationLanguage(ViewDeclarationLanguageFactoryImpl.java:54)
org.apache.myfaces.application.ViewHandlerImpl.getViewDeclarationLanguage(ViewHandlerImpl.java:184)
jakarta.faces.application.ViewHandlerWrapper.getViewDeclarationLanguage(ViewHandlerWrapper.java:149)
org.apache.myfaces.application.ViewIdSupport.isViewExistent(ViewIdSupport.java:519)
org.apache.myfaces.application.ViewIdSupport.handleSuffixMapping(ViewIdSupport.java:450)
org.apache.myfaces.application.ViewIdSupport.deriveViewId(ViewIdSupport.java:128)
org.apache.myfaces.application.ViewIdSupport.deriveLogicalViewId(ViewIdSupport.java:101)
org.apache.myfaces.application.ViewHandlerImpl.deriveLogicalViewId(ViewHandlerImpl.java:123)
jakarta.faces.application.ViewHandlerWrapper.deriveLogicalViewId(ViewHandlerWrapper.java:128)
org.apache.myfaces.lifecycle.RestoreViewExecutor.execute(RestoreViewExecutor.java:223)
org.apache.myfaces.lifecycle.LifecycleImpl.executePhase(LifecycleImpl.java:172)
org.apache.myfaces.lifecycle.LifecycleImpl.execute(LifecycleImpl.java:125)
jakarta.faces.webapp.FacesServlet.service(FacesServlet.java:223)
com.ibm.ws.webcontainer.servlet.ServletWrapper.service(ServletWrapper.java:1260)
com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:748)
com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:445)
com.ibm.ws.webcontainer.filter.WebAppFilterChain.invokeTarget(WebAppFilterChain.java:195)
com.ibm.ws.webcontainer.filter.WebAppFilterChain.doFilter(WebAppFilterChain.java:100)
io.openliberty.org.apache.faces40.fat.literals.AttributeFilter.doFilter(AttributeFilter.java:38)
com.ibm.ws.webcontainer.filter.FilterInstanceWrapper.doFilter(FilterInstanceWrapper.java:203)
com.ibm.ws.webcontainer.filter.WebAppFilterChain.doFilter(WebAppFilterChain.java:93)
com.ibm.ws.webcontainer.filter.WebAppFilterManager.doFilter(WebAppFilterManager.java:1071)
com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1209)
com.ibm.ws.webcontainer.webapp.WebApp.handleRequest(WebApp.java:5080)
```

It looks to be related to creating a `@View` but I figured that was outside the scope of this test.